src/components/ContactInfo/ContactInfo.tsx

```diff
@@ -11,27 +33,32 @@ const ContactInfo = (props: ContactInfoProps) => {
           <div className={'grid contact-info__grid__element'}>
             <img src="/assets/medicon/images/contact-info-1.png" alt="address"/>
             <div>
-              <p>MEDICON a.s. Antala Staška 1670/80 140 00 Praha 4</p>
+              {address && <p>{address}</p>}
             </div>
           </div>
           <div className={'grid contact-info__grid__element'}>
             <img src="/assets/medicon/images/contact-info-2.png" alt="phone number"/>
             <div>
-              <a href={'tel:+420 261 006 111'}>tel.: +420 261 006 111</a><br/>
-              <a href={'tel:+420 261 006 210'}>fax: +420 261 006 210</a>
+              {phones && phones.map((item, i) => (
```

Comment:
use for descriptive name for variable in mapping, if you  are iteraring phones then call it phone.